app/javascript/components/pages/ReviewNew.js

```diff
@@ -0,0 +1,90 @@
+import React, { Component } from 'react'
+import { Form, FormGroup, Label, Input, Button } from 'reactstrap'
+import { Redirect } from 'react-router-dom'
+
+
+class ReviewNew extends Component {
+    constructor(props) {
+        super(props)
+        this.state = {
+            newReview: {
+                title: "",
+                date_posted: "",
+                review_text: "",
+                user_id: this.props.current_user.id,
+                school_id: ""
```

Comment:
This should be updated more dynamically than the user needing to know the school id they want this attached to.